TIG/TIG/Feature/Home/HomeViewModel.swift

```diff
@@ -18,11 +19,22 @@ final class HomeViewModel {
         
         // TimelineView
         var isEditMode: Bool = false
-        //var dailyContents: [DailyContent] = TestData.dailycontents
+        var dailyContents: [DailyContent] = TestData.dailycontents
         var timelines: [Timeline] = TestData.dailycontents[0].timelines
 
         // RepeatEditView
         var selectedDay: Day = .sun
+        
+        // TimerView
+        var counter: Int = 0
+        var countTo: Int {
+            if let selectedContent = dailyContents.first(where: { Calendar.current.isDate($0.date, inSameDayAs: currentDate) }) {
+                return selectedContent.totalAvailabilityTime
+            } else {
+                return 0
+            }
+        }
+        var timer: AnyCancellable?
     }
```

Comment:
timer와 관련된 변수들로 보이는데, 이는 View의 상태와는 관련있는 변수로 보이지 않습니다.
실제로 TimerView에서도 counter, countTo 변수를 사용하고 있지 않고 HomeViewModel에서만 사용하고 있고여.
이를 State 밖으로 빼주면서 private으로 설정해주는 것이 좋아보입니다.
```suggestion
struct State {
...
}
// TimerView
private var counter: Int = 0
private var countTo: Int {
    if let selectedContent = dailyContents.first(where: { Calendar.current.isDate($0.date, inSameDayAs: currentDate) }) {
        return selectedContent.totalAvailabilityTime
    } else {
        return 0
    }
}
var timer: AnyCancellable?

```

Reply:
했음요